src/main/java/calculator/controller/CalculatorController.java

```diff
@@ -0,0 +1,25 @@
+package calculator.controller;
+
+import calculator.model.CalculatorModel;
+import calculator.view.CalculatorView;
+
+public class CalculatorController {
+    private final CalculatorModel calculatorModel;
+    private final CalculatorView calculatorView;
+
+    public CalculatorController() {
+        calculatorModel = new CalculatorModel();
+        calculatorView = new CalculatorView();
+    }
+
+    public void calStart() {
+        String[] digits = calculatorView.getDigits();
+        calResult(calculatorModel.getResult(digits));
```

Comment:
```suggestion
        calculatorView.printResult(calculatorModel.getResult(digits));
```

이 형태가 **결과를 출력한다**라고 더 직관적으로 읽힐 것 같습니다